modules/ui/raw_tag_editor.js

```diff
@@ -282,8 +282,9 @@ export function uiRawTagEditor(context) {
 
         function isReadOnly(d) {
             // Read-only for source=digitalglobe on ML roads. TODO: switch to check on __fbid__
-            if (_entityID && _entityID.indexOf('w-') === 0 && d.key === 'source' && d.value === 'digitalglobe') {
-                return true;
+            if (_entityID && _entityID.indexOf('w-') === 0 && d.key === 'source' 
+                && (d.value === 'digitalglobe' || d.value === 'maxar')) {
+                    return true;
```

Comment:
I think it's OK to unindent `return true;` here; e.g. imaging the case we had more lines of codes within the block, we wouldn't want to further-indent the whole block in that case.